lte/gateway/fabfile.py

```diff
@@ -215,122 +235,138 @@ def open_orc8r_port_in_vagrant():
         ";next}1' Vagrantfile >> Vagrantfile.bak00 && " \
         "cp Vagrantfile.bak00 Vagrantfile && rm Vagrantfile.bak00"
 
-    local("{} || ({})".format(cmd_yes_if_exists, cmd_insert_line))
+    c.run(f"{cmd_yes_if_exists} || ({cmd_insert_line})")
```

Comment:
This command needs to be executed on the host and not in a VM. (Actually the Vagrantfile is modified before a VM is started). Is c.run doing this?

Reply:
Yes, `c` is the Connection to the local host, i.e. `c.run` executes command on the local host.